src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -42,6 +44,14 @@ export class MultiSelectComponent<T> implements OnInit, ControlValueAccessor {
 
   constructor() {}
 
+  @HostListener('click', ['$event'])
+  protected stopEventPropagation(): void {
+    this.isStopOptionsClosing = true
```

Comment:
I am not entirely sure what `isStopOptionsClosing` is indicating :)
From the logic based on it, I assume that it is whether the component `hasBeenClicked`.
If that is the case, then the method could also be renamed to something like `registerClick`.

Reply:
Renamed. The idea behind this variable is to let the click event propagate (to can close all another open dropdowns), but still if the click is on the current component not to close the dropdown.

Reply:
Thanks, that makes sense.

I might not have given the best description. My suggestion was to rename `isStopOptionsClosing` to `hasBeenClicked` and `stopEventPropagation` to `registerClick`.